front-end/src/components/Post/Poll/CouncilSignals.tsx

```diff
@@ -20,39 +19,42 @@ import getEncodedAddress from '../../../util/getEncodedAddress';
 interface Props {
 	className?: string
 	data?: PostVotesQuery | undefined
+	pollBlockNumberEnd?: number | null | undefined
 }
 
-const CouncilSignals = ({ className, data }: Props) => {
+const CouncilSignals = ({ className, pollBlockNumberEnd, data }: Props) => {
 	const [ayes, setAyes] = useState(0);
 	const [nays, setNays] = useState(0);
 	const [memberSet, setMemberSet] = useState<Set<string>>(new Set<string>());
 	const [councilVotes, setCouncilVotes] = useState<OffchainVote[]>([]);
-	const { api, isApiReady } = useContext(ApiPromiseContext);
+	const currentBlockNumber = useGetCurrentBlockNumberQuery();
+	const councilAtPollEndBlockNumber = useCouncilAtBlockNumberQuery({ variables: { blockNumber: pollBlockNumberEnd || 0 } });
+	const councilAtCurrentBlockNumber = useCouncilAtBlockNumberQuery({ variables: { blockNumber: currentBlockNumber?.data?.blockNumbers?.[0]?.number || 0 } });
```

Comment:
Same here, we shouldn't make the query if we don't have the right current block number.

Reply:
useCouncilAtBlockNumberQuery can't be called conditionally